views/vulns.erb

```diff
@@ -0,0 +1,156 @@
+<h1>Vulnerabilities</h1>
+
+<form id="recon" action="/vulns" method="post">
+  <div class="field">
+    <label class="label is-required">URL</label>
+
+    <div class="control">
+      <% if @errors && @errors[:url] %>
+        <input class="input is-danger" name="url" required><%=h params[:url] %></input>
+
+        <% @errors[:url].each do |error| %>
+        <p class="help is-danger"><%=h error %></p>
+        <% end %>
+      <% else %>
+        <input class="input" name="url" placeholder="https://example.com" required><%=h params[:url] %></input>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="field">
+    <button type="submit" class="button is-primary">Scan</button>
+  </div>
+
+  <div class="advanced">
+    <a class="advanced-toggle">Advanced Options</a>
+
+    <div class="advanced-content">
+      <h3>LFI</h3>
+
+      <div class="field">
+        <label class="label">Os</label>
+
+        <div class="control">
+          <select class="select" name="lfi[os]">
+            <option value="unix">unix</option>
+            <option value="windows">windows</option>
```

Comment:
I think we should add the above code to add an optional `selected` attribute if the param is set.

```erb
<%= " checked" if params.dig(:lfi, :os)  == 'windows' %>
```